### ISSUE TITLE
Bot self dismiss request changes review on VIP approval

### ISSUE THREAD

**Issue by davidsword** [Automattic/vip-go-ci#136]
After VIP approves a PR, VIP then needs to manually dismissing any bots 'request changes' review to unblock the PR.

It would be nice if the bot could self dismiss its own reviews with something like 'superseded by VIP review' after an approval by VIP staff.

**Comment by gudmdharalds** [comment]
This is now on our roadmap.
